src/site/resources/styleguides/google-java-style-20180523/javaguide.html

```diff
@@ -259,14 +261,16 @@ <h4 id="s4.1.1-braces-always-used">4.1.1 Braces are used where optional</h4>
 <code class="prettyprint lang-java">while</code> statements, even when the
 body is empty or contains only a single statement.</p>
 
+<p>Other optional braces, such as those in a lambda expression, remain optional.</p>
+
```

Comment:
Will be handled by: https://github.com/checkstyle/checkstyle/issues/14908

Reply:
This issue is merged/closed.